portfolio/src/main/webapp/index.html

```diff
@@ -10,28 +10,32 @@
     <script src="https://kit.fontawesome.com/96f9ad965c.js" crossorigin="anonymous"></script>
   </head>
   <body>
-    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
-      <a class="navbar-brand" href="#">Portfolio</a>
-      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
-        <span class="navbar-toggler-icon"></span>
-      </button>
-      <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
-        <div class="navbar-nav">
-          <a class="nav-item nav-link active" href="#">About me<span class="sr-only">(current)</span></a>
-          <a class="nav-item nav-link" href="#">Projects</a>
-          <a class="nav-item nav-link" href="#">Gallery</a>
-          <a class="nav-item nav-link" href="#">Fun</a>
+    <div id="content">
+      <div class="media my-3">
+        <img src="./images/me.jpg" alt="Picture of Shaar" class="img-thumbnail" id="profile-picture">
+        <div class="media-body p-2 ml-2" id="about-me">
+          <h3 class="mt-0">Mariano Shaar</h5>
```

Comment:
closing element (/h5) does not correspond with opening element (h3)

Reply:
Done!